### ISSUE TITLE
Question about JoinAP

### ISSUE THREAD

**Issue by kalvinchang** [thu-spmi/CAT#62]
Hi. I apologize if this is not the right place to post a question about JoinAP, but I wasn't sure where to post it.

I have a few questions about JoinAP from Zhu et al 2021 (https://arxiv.org/pdf/2107.05038.pdf).

* Where does the model obtain the phones in Figure 2? Are the phones obtained from the ground truth transcriptions or are they first predicted by the acoustic model?
* By top-down, are you referring to breaking phones down into articulatory phonetic features using panphon?
* During test time, are the phonetic transcriptions generated by Phonetisaurus also fed into the acoustic model as phone sequences? If not, where do the phones come from?

Thank you in advance!!

**Comment by ZhijianOu** [comment]
- In Figure 2(a), phone denotes hypothesized phone. At frame t, for each phone i, we use Eq. (2) or (3) to calculate the logits, which are then used to calculate the phone (posterior) probabilities for CTC-CRF or CTC. This is explained in the paragraph under Eq. (7).

- >By top-down, are you referring to breaking phones down into articulatory phonetic features using panphon?

Partly Yes.
breaking phones into articulatory features, and then calculating the phone embeddings based on the articulatory features.

- In testing, we use WFST-based decoding based on the WFST composed from T, L, and G. L is the WFST that maps a word into a phone sequence. T and G represent the CTC topology and n-gram language model, respectively.

**Comment by kalvinchang** [comment]
I see. Thank you!

**Comment by ZhijianOu** [comment]
>It is only during decoding that you pick a phone for time t using the posterior phone probability distribution.

Yes if you use the best path decoding.
